docs/zh/latest/plugins/skywalking-logger.md

```diff
@@ -23,9 +23,9 @@ title: skywalking-logger
 
 ## 描述
 
-`skywalking-logger` 是一个插件，可将 Access Log 数据通过 `HTTP` 推送到 SkyWalking OAP 服务器。如果上下文中存在 `tracing context`，插件会自动建立 `trace` 与日志的关联，并依赖于 [SkyWalking Cross Process Propagation Headers Protocol](https://skywalking.apache.org/docs/main/latest/en/protocols/skywalking-cross-process-propagation-headers-protocol-v3/) 。
+`skywalking-logger` 是一个可将 Access Log 数据通过 `HTTP` 推送到 `SkyWalking OAP` 服务器的插件。如果上下文中存在 `tracing context`，插件会自动建立 `trace` 与日志的关联，这有赖于 [SkyWalking Cross Process Propagation Headers Protocol](https://skywalking.apache.org/docs/main/latest/en/protocols/skywalking-cross-process-propagation-headers-protocol-v3/)的支持 。
 
-这将提供将 Access Log 数据作为 JSON 对象发送到 SkyWalking OAP 服务器的功能。
+本插件提供将 Access Log 数据作为JSON对象发送到 `SkyWalking OAP` 服务器的功能。
```

Comment:
```suggestion
本插件提供将 Access Log 数据作为 JSON 对象发送到 `SkyWalking OAP` 服务器的功能。
```

Reply:
We need a black space between Chinese and English, you could refer to https://github.com/sparanoid/chinese-copywriting-guidelines.